### ISSUE TITLE
"Get started with examples" page

### ISSUE THREAD

**Issue by life-efficient** [life-efficient/Conversationally#63]
When a user first logs in, they should be presented with a series of good examples of agents that do different things to play around with.

- Show 5 different cards, each with an avatar, name, and description (use material UI to make it look sexy)
- Have another card for "Design one from scratch" which should open up the customisation page (by rendering a `Navigate` component (depends on query string customisation being implemented) (might be worth making the customisation page full-screen).

**Comment by life-efficient** [comment]
Depends on #64 and #65
